x-pack/plugins/security_solution/server/endpoint/services/artifacts/manifest_manager/manifest_manager.ts

```diff
@@ -266,29 +324,23 @@ export class ManifestManager {
    * Blocklist list
    * @protected
    */
-  protected async buildBlocklistArtifacts(): Promise<ArtifactsBuildResult> {
+  protected async buildBlocklistArtifacts(allPolicyIds: string[]): Promise<ArtifactsBuildResult> {
     const defaultArtifacts: InternalArtifactCompleteSchema[] = [];
-    const policySpecificArtifacts: Record<string, InternalArtifactCompleteSchema[]> = {};
     const buildArtifactsForOsOptions: BuildArtifactsForOsOptions = {
       listId: ENDPOINT_BLOCKLISTS_LIST_ID,
       name: ArtifactConstants.GLOBAL_BLOCKLISTS_NAME,
     };
 
-    for (const os of ArtifactConstants.SUPPORTED_EVENT_FILTERS_OPERATING_SYSTEMS) {
+    for (const os of ArtifactConstants.SUPPORTED_BLOCKLISTS_OPERATING_SYSTEMS) {
```

Comment:
🦅 👁️

Reply:
This was there since last year! 🙀  Thankfully it is the same set of OSs. 😅